### PR TITLE
Forms back end

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by johngian** [comment]
I am getting this error on `/activities`:

`Reverse for 'fxosprice_new' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []`

File: marketpulse/main/templates/fxosprice_new.html [hunk 1]
```diff
@@ -8,33 +8,41 @@
...
+            <label for="id_is_online">Is this an online shop?</label>
             <label class="pack-switch">
-              <input type="checkbox" id="onlineshop-check">
+              {{ location_form.is_online }}
               <span></span>
             </label>
           </div>
           <div class="form-group">
-            <input type="text" name="shop_name" placeholder="Shop Name" required>
+            {{ location_form.shop_name }}
```
Comment (comzeradd) [inline marketpulse/main/templates/fxosprice_new.html]: Can you add a `Shop Name` placeholder?

File: marketpulse/base/static/css/app.css [hunk 2]
```diff
@@ -73,4 +80,3 @@ fieldset legend {
 
 #colophon a:hover, #colophon a:active, #colophon a:focus {
   color: #76D3FF !important;
-}
```
Comment (comzeradd) [inline marketpulse/base/static/css/app.css:188]: Probably accidentally you deleted the closing bracket here.

**Comment by johngian** [comment]
The error messages need some UI love but let's not block this PR with that.

File: marketpulse/base/templates/form-error.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{% if field.errors %}
+  {% if field.errors is string %}
+      <small class="error">
+        {{ field.errors }}
+      </small>
+  {% else %}
```
Comment (comzeradd) [inline marketpulse/base/templates/form-error.html:6]: This seems over-indented. We use 2 spaces on templates.

**Comment by akatsoulas** [comment]
@johngian pr updated. I removed completely the `activities` url since it's no used anywhere right now. We can add it when we have the rest of the UI.

**Comment by akatsoulas** [comment]
@comzeradd final r?

**Comment by johngian** [comment]
LGTM r+ :rocket:

**Pull request merged**